build-system/compile/bundles.config.extensions.json

```diff
@@ -157,10 +157,37 @@
   {"name": "amp-facebook-page", "version": "0.1", "latestVersion": "0.1"},
   {
     "name": "amp-fit-text",
-    "version": ["0.1", "1.0"],
+    "version": "0.1",
     "latestVersion": "0.1",
     "options": {"hasCss": true}
   },
+  {
+    "name": "amp-fit-text",
+    "version": "1.0",
+    "latestVersion": "0.1",
+    "options": {
+      "hasCss": true,
+      "binaries": [
+        {
+          "entryPoint": "component.js",
+          "outfile": "component-preact.js",
+          "external": ["preact", "preact/dom", "preact/compat", "preact/hooks"],
```

Comment:
Would the `external` and `remap` be repeated each time mostly unchanged? If so, maybe some sort of a template would make it easier?